src/osm_configurator/model/project/data_visualizer.py

```diff
@@ -1,7 +1,12 @@
 from __future__ import annotations
 
 import src.osm_configurator.model.project.configuration.cut_out_configuration
-import matplotlib
+
+import pathlib
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from src.osm_configurator.model.project.configuration.cut_out_configuration import CutOutConfiguration
 
 
 class DataVisualizer:
```

Comment:
this class changed a lot on the model branch, because i implemented it fully, be carefull when merging that you use the correct implementation.